Menu.cpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by alexteper on 20/7/20.
+//
+
+#include "Menu.h"
+
```

Comment:
Poner los `includes` del .h aca
```suggestion
#include <iostream>
#include "MenuAeropuerto.h"
#include "MenuVuelo.h"
```